main.tf

```diff
@@ -276,6 +276,10 @@ resource "aws_lambda_function" "firehose_lambda_transform" {
   timeout                        = var.lambda_function_timeout
   reserved_concurrent_executions = var.lambda_reserved_concurrent_executions
 
+  environment  {
```

Comment:
Looks like this should be `environment {` and not `environment  {` 
(see the space)